src/app/(main)/page.tsx

```diff
@@ -0,0 +1,10 @@
+import type { Metadata } from "next"
+import { IndexPage } from "../../components/index-page"
+
+export const metadata: Metadata = {
+  title: "GraphQL | A query language for your API",
+}
+
+export default function Page() {
+  return <IndexPage />
```

Comment:
While we're at it... WDYT about moving `components/index-page` to this file? I don't know how that happened. Might be that people thought only `.mdx` files are supported for pages.

Reply:
There's a load of files and folders for the index-page, I appreciate that it's well-organized. I think we should leave it as-is for now, just to get this over the line. We should revisit the component organization later.

Reply:
fair dos